scripts/pages/pgFavorites.ts

```diff
@@ -131,8 +118,25 @@ export default class PgFavorites extends withDismissAndBackButton(PgFavoritesDes
         this.lvMain.itemCount = this.data.length;
         if (this.data && this.data.length > 0) {
             if (this.data[0].type === 'LVI_EMPTY_ITEM') {
+                this.lvMain.onRowSelected = (item, index: number) => {
```

Comment:
You should move this event into `initListView` function. Because `initListView` function will trigger single one time but refreshListView can trigger more than one. Secondly, you should define just one onRowSelected event and you can add condition in onRowSelected. Like this

```ts
this.lvMain.onRowSelected = (item, index: number) => {
   if (item.type === 'LVI_EMPTY_ITEM') {
        // ....
   } else {
        // ...
    }
}
```

Reply:
MISSING.